CONTRIBUTING.md

```diff
@@ -109,7 +109,7 @@ If your architecture decision is a simple change, you may contribute directly wi
 
 In certain circumstances, the architecture decision may require changes to the ICS spec. Note that the spec is responsible for defining language-agnostic, implementation-agnostic behaviors for the ICS protocol. Whereas ADRs are responsible for communicating implementation decisions contained within this repo.
 
-To create an ADR, follow the [template](docs/docs/adrs/adr-template.md) and [doc](docs/docs/adrs/intro.md). If you would like to see examples of how these are written, please refer to the current [ADRs](docs/docs/adrs).
+To create an ADR, follow the [template](docs/docs/adrs/_adr-template.md) and [doc](docs/docs/adrs/intro.md). If you would like to see examples of how these are written, please refer to the current [ADRs](docs/docs/adrs).
```

Comment:
Breaks markdown links, I don't understand why you did this

Reply:
The docs will not deploy with broken links

Reply:
I wanted to avoid rendering this page. I was hoping (for some reason) that files that start with `_` are not rendered.

Reply:
@MSalopek any ideas how to fix this?

Reply:
There's the exclude option that goes into `docusaurus.config.js`
* https://docusaurus.io/docs/api/plugins/@docusaurus/plugin-content-docs#exclude


See commit: https://github.com/cosmos/interchain-security/pull/1937/commits/99d7d90fcf0abc758f34970e8b30833a2a9058c4 

```diff
  // TLDR:
  presets: [
    [
      "classic",
      /** @type {import('@docusaurus/preset-classic').Options} */
      ({
        docs: {
          routeBasePath: "/",
          sidebarPath: require.resolve("./sidebars.js"),
          ....
          },
+          exclude: ["**/templates/*"],
        },
```